crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -298,18 +298,18 @@ impl Client {
     }
 
     /// Encrypt and upload the file to be read from `reader` and construct an
-    /// attachment message with `body`, `content_type`, `info` and `thumbnail`.
+    /// attachment message with `body`, `content_type`, `info` and `thumbnail`,
+    /// optionanly with `formatted` and `filename`.
```

Comment:
This comment needs to be tweaked too, since `thumbnail` and `formatted` aren't present anymore.